src/mlpack/core/dists/gamma_distribution.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file gamma_distribution.cpp
+ * @author Yannis Mentekidis
+ *
+ * Implementation of the methods of GammaDistribution.
+ */
+#include "gamma_distribution.hpp"
+
+#include <boost/math/special_functions/digamma.hpp>
+#include <boost/math/special_functions/trigamma.hpp>
```

Comment:
The travis build can't find this function. Why? Isn't boost::math a dependency of mlpack?

Or is it a new function added to a later version of boost?